packages/lexical-react/src/LexicalDraggableBlockPlugin.tsx

```diff
@@ -43,17 +43,56 @@ const Downward = 1;
 const Upward = -1;
 const Indeterminate = 0;
 
-let prevIndex = Infinity;
+interface ElementInfosHolder {
```

Comment:
This name is kinda redundant, information doesn't have a separate plural form in english and generally speaking any data structure is a "holder" (particularly in a language like JS) so "InfosHolder" doesn't make sense. I would just call it `BlockElement` because that's how you're referring to it in the API (e.g. `getBlockElementFromNodes` returns this type)

```suggestion
interface BlockElement {
```